featuretools/tests/conftest.py

```diff
@@ -565,3 +565,8 @@ def divide_by_zero_es_koalas(divide_by_zero_es_pd):
                          index=df.ww.index,
                          logical_types=df.ww.logical_types)
     return es
+
+
+@pytest.fixture
+def rolling_series():
```

Comment:
what if we kept the "rolling_series_pd" name to avoid having to check the definition to see if its a pandas only fixture or a parameterized one?

Reply:
sure, happy to keep as `rolling_series_pd`